lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -389,7 +389,7 @@ def add_creditcard(post, creditcard, options)
 
           if creditcard.is_a?(NetworkTokenizationCreditCard)
             card[:cryptogram] = creditcard.payment_cryptogram
-            card[:eci] = creditcard.eci
+            card[:eci] = creditcard.eci.rjust(2, '0') if creditcard.eci =~ /^[0-9]+$/
```

Comment:
Yes, this contains another change wherein it is not sent if non-numeric. 

I dug though the history looking at _why_ we allow setting `eci` to `"recurring"` to flip the recurring option, and it seems to just be some legacy and should eventually be dropped.